api/python/cell_census/src/cell_census/get_anndata.py

```diff
@@ -54,7 +57,7 @@ def get_anndata(
     exp = get_experiment(census, organism)
     with exp.axis_query(
         measurement_name,
-        obs_query=soma.AxisQuery(value_filter=obs_value_filter) if obs_value_filter is not None else soma.AxisQuery(),
-        var_query=soma.AxisQuery(value_filter=var_value_filter) if var_value_filter is not None else soma.AxisQuery(),
+        obs_query=soma.AxisQuery(value_filter=obs_value_filter, coords = obs_coords),
+        var_query=soma.AxisQuery(value_filter=var_value_filter, coords = var_coords),
```

Comment:
@atolopko-czi unless I missed something the `if` wasn't needed because the defaults `value_filter=None` and `coords=(slice(None),)` are properly handled by `AxisQuery`